velox/exec/fuzzer/AggregationFuzzerBase.cpp

```diff
@@ -318,8 +329,21 @@ std::vector<RowVectorPtr> AggregationFuzzerBase::generateInputDataWithRowNumber(
           generator->generate(signature.args, vectorFuzzer_, rng_, pool_.get());
     }
 
+    // Randomly generate the number of partitions
+    auto numPartitions = randInt(1, maxPartitions);
```

Comment:
Hi @pramodsatya, we previously determine maxPartitions to be a random number between size and size/3, and now determine numPartitions to be a random number between 1 and maxPartitions. Then, could we combine these two steps to just choose numPartitions to be a random number between 1 and size?

Reply:
Thanks for the suggestion @kagamiori, save for the corner case where `size = 0` I don't think `maxPartitions` is necessary and `numPartitions` can be updated as you suggested. Added check for `size = 0` and updated `numPartitions`. Could you please take another look?